sys/net/gnrc/netif/gnrc_netif.c

```diff
@@ -40,6 +40,16 @@
 
 #define _NETIF_NETAPI_MSG_QUEUE_SIZE    (8)
 
+#ifdef MODULE_GNRC_NETIF_EVENTS
+/**
+ * @brief   Event type used for passing netdev pointers together with the event
+ */
+typedef struct {
+    event_t super;
+    netdev_t *dev;
+} event_netdev_t;
```

Comment:
If you don't want to make this type public, I'd prefer to prefix it with `_` to make the usage code easier to follow.